src/mmw/js/src/compare/controllers.js

```diff
@@ -4,9 +4,16 @@ var _ = require('lodash'),
     App = require('../app'),
     router = require('../router').router,
     views = require('./views'),
-    modelingModels = require('../modeling/models.js');
+    modelingModels = require('../modeling/models.js'),
+    modelingControls = require('../modeling/controls');
+
+var synchronizer = modelingControls.PrecipitationSynchronizer;
```

Comment:
That's odd, because we do use this in other places, like this: https://github.com/WikiWatershed/model-my-watershed/blob/develop/src/mmw/js/src/modeling/controllers.js#L8

I wonder if it's because we're not using the `new` keyword, or because of the [IIFE](https://en.wikipedia.org/wiki/Immediately-invoked_function_expression).

But it's not a big deal. I don't mind if we leave it.